core/src/databases/csv.rs

```diff
@@ -87,7 +87,7 @@ impl UpsertQueueCSVContent {
             let slug = if curr == "__dust_id" {
                 curr.to_string()
             } else {
-                let s = Self::slugify(curr);
+                let s = Self::slugify(&curr.to_lowercase());
```

Comment:
should lowercasing be done by slugify instead?>

Reply:
Well slugify does support camelCase processing... the goal here is to mimick front as much as possible.